src/DwitTech.NotificationService.WebApi/Controllers/EmailController.cs

```diff
@@ -1,11 +1,52 @@
 ﻿using DwitTech.NotificationService.Core.Dtos;
 using DwitTech.NotificationService.Core.Interfaces;
+using DwitTech.NotificationService.Core.Services;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 
 namespace DwitTech.NotificationService.WebApi.Controllers
 {
     public class EmailController : BaseController
     {
-        
+        private readonly IEmailService _emailService;
+        public EmailController(IEmailService emailService)
+        {
+            _emailService = emailService;
+        }
+
+        [AllowAnonymous]
+        [HttpPost]
+        public IActionResult SendEmail([FromBody] EmailDto MailMessage)
+        {
+
+            var createResult = _emailService.CreateEmail(MailMessage);
```

Comment:
too much logic here. It should just call a sendEmail method in emailservice that will save to db and send the email.